CHANGES.md

```diff
@@ -10,6 +10,7 @@
 - Added checks for supported 3D Tiles extensions. [#9552](https://github.com/CesiumGS/cesium/issues/9552)
 - Added documentation clarifying that the `outlineWidth` property will be ignored on all major browsers on Windows platforms. [#9600](https://github.com/CesiumGS/cesium/pull/9600)
 - Added documentation for `KmlTour`, `KmlTourFlyTo`, and `KmlTourWait`. Added documentation and a `kmlTours` getter to `KmlDataSource`. Removed references to `KmlTourSoundCues`. [#8073](https://github.com/CesiumGS/cesium/issues/8073)
+- Added the ability for users to turn of outlines for 3D models. [#8959](https://github.com/CesiumGS/cesium/issues/8959)
```

Comment:
I think OSM should be highlighted here since it's the main thing using [CESIUM_primitive_outline](https://github.com/KhronosGroup/glTF/tree/master/extensions/2.0/Vendor/CESIUM_primitive_outline).

"Added the ability to hide outlines on OSM Buildings and other tilesets and glTF models using the [CESIUM_primitive_outline](https://github.com/KhronosGroup/glTF/tree/master/extensions/2.0/Vendor/CESIUM_primitive_outline) extension. [#8959](https://github.com/CesiumGS/cesium/issues/8959)"